### DISCUSSION TITLE
Error 3271 (full disk) but have enough space

### DISCUSSION THREAD

**Discussion by WJvE** [olahallengren/sql-server-maintenance-solution#395] [Questions & Answers]
Hi Ola,

I've been stuck for weeks with a backup issue for one DB. We use your scripts for years without any problems
Since april 2020 one DB failed all the time for type FULL and DIFF backup. This DB has TDE encryption and that was not a problem before april. The verify backup will also fail..

In the commandlog, the executed query is:
BACKUP DATABASE [DBNAME] TO DISK = N'\\\share\folder1\folder2\folder3\FULL\BACKUPNAME.bak' WITH CHECKSUM, NO_COMPRESSION

This will fail when the job use the scripts to backup the DB. But if I execute the query above manually, it will succeeds! I really don't know why and our system operator who manage the SAN didn't know either. Last weekend, I've switched from primary to secondary SAN to test if that will help but no.
Still the same error.

The errors in the log scripts are:
Msg 3271, Sev 16, State 1, Line 1 : A nonrecoverable I/O error occurred on file "\\\share\folder1\folder2\folder3\FULL\BACKUPNAME.bak:" 112(There is not enough space on the disk.). [SQLSTATE 42000]
Msg 3013, Sev 16, State 1, Line 1 : BACKUP DATABASE is terminating abnormally. [SQLSTATE 42000]

The DB is 6,5 GB and there is > 1TB free space on the SAN.

Can you help us out?

Thanks in advance!

Sincerly,
Willem Jan

**Comment by olahallengren** [comment]
@WJvE, it should not make any difference if you execute the command from my stored procedure, or as a standalone command.
